project/src/components/main-page/main-page.tsx

```diff
@@ -0,0 +1,199 @@
+                height="327"
+              />
+            </div>
+
+            <div className="film-card__desc">
+              <h2 className="film-card__title">{title}</h2>
+              <p className="film-card__meta">
+                <span className="film-card__genre">{genre}</span>
+                <span className="film-card__year">{year}</span>
+              </p>
+
+              <div className="film-card__buttons">
+                <button className="btn btn--play film-card__button" type="button">
+                  <svg viewBox="0 0 19 19" width="19" height="19">
+                    <use xlinkHref="#play-s"></use>
+                  </svg>
+                  <span>Play</span>
+                </button>
+                <button className="btn btn--list film-card__button" type="button">
+                  <svg viewBox="0 0 19 20" width="19" height="20">
+                    <use xlinkHref="#add"></use>
+                  </svg>
+                  <span>My list</span>
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <div className="page-content">
+        <section className="catalog">
+          <h2 className="catalog__title visually-hidden">Catalog</h2>
+
+          <ul className="catalog__genres-list">
+            <li className="catalog__genres-item catalog__genres-item--active">
+              <a href="#" className="catalog__genres-link">All genres</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">Comedies</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">Crime</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">Documentary</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">Dramas</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">Horror</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">Kids & Family</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">Romance</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">Sci-Fi</a>
+            </li>
+            <li className="catalog__genres-item">
+              <a href="#" className="catalog__genres-link">Thrillers</a>
+            </li>
+          </ul>
+
+          <div className="catalog__films-list">
+            {renderFilmCards().map((card) => card)}
```

Comment:
```suggestion
            {new Array(20).fill(0).map((_, i) => <FilmCard key={i} />) /* eslint-disable-line react/no-array-index-key */}
```

Можно отказаться от `renderFilmCards` и на ходу сгенерировать массив из нужного количества элементов.
Не знаю, рассказывали ли уже про обязательный проп `key` для рендера компонентов через циклы. Если нет, то он нужен для оптимизаций перерисовки, на следующих лайвах расскажут. Индекс для этого использовать запрещено линтером, но так как у нас сейчас реальных идентификаторов нет, можно просто заглушить линтер. В будущем это место перепишется на реальные данные.